packages/odyssey-design-tokens/src/font.json

```diff
@@ -11,7 +11,6 @@
       }
     },
     "size": {
-      "base": { "value": "16px" },
```

Comment:
This is one property we know all consumers are expecting to theme. Why are we removing it?

Reply:
I was concerned about the hardcoded pixel value hurting accessibility for low sighted users who use a larger browser default font size. I think we talked thru a couple ways to change font sizing in slack before? One way is to change the `html` font-size in user provided global CSS, another is using our theme API to target say `FontSizeBody`.

Reply:
As long as the rest of the UI is set in `rem`, device/browser settings should still play nice. Ex: Zooming on a page will still increase the base value of `rem`. If we want to be doubly safe, we could drop the units and use a `calc()` to generate a relative size based on the delta between the `16px` default at `100%` and the themed size. 

Ex: 

```
html { 
  font-size: calc((var(--FontSizeBase) / 16) * 100%);
}
```

Re: user provided global CSS, I think the expectation is that modifying the base font size will be consistent with theming other properties. If that won't be the case, we should communicate this loudly. In all other cases we are asking teams to eschew as much global CSS as possible, so this would likely be a surprise.

Re: targeting individual sizes, this feels like the opposite of what most folks would expect. The promise of our rem-based UI to Design is something like, "change one value and the UI will scale to meet your needs." If we ask users to target the size palette directly, they will need recalculate new `rem` values based on their new base size across the board.

Reply:
Maybe a short term solution is a single-property injection into `html` handled by `odyssey-react-theme`?

That way, `odyssey-react` itself doesn't own anything in the global space, and the caveat of Odyssey touching `html` becomes a consideration for those who need theming.

Reply:
@edburyenegren-okta if this token is referenced directly in a style module (as it was before my change here) then the declaration value will be `16px` at run time and wont' behave like `rem`. Are you saying there is another behavior happening here?

Reply:
The changes to the modules are 💯 . No component module should be referencing this value directly, instead preferring `FontSizeBody` (or similar) as you've elected to use.

This token/value is only intended to be used as a referent for `rem` values used within the component modules. It's only intended application would be on `html` in order to set that reference size.

Reply:
@edburyenegren-okta great I think what we need then is more scripting to filter out this property from being created as a token. If its availble, people will use it. I can add that in this PR